app/templates/suppliers/create_new_supplier.html

```diff
@@ -19,26 +19,27 @@
 
 <div class="start-page">
   {%
-    with heading = "Create supplier account"
+    with heading = "Create a supplier account",
+         smaller = True
   %}
     {% include "toolkit/page-heading.html" %}
   {% endwith %}
   <div class="grid-row">
     <div class="column-two-thirds">
-      <div class="check-list">
-        <p class="lead">To create a supplier account, you’ll need your company’s:</p>
-        <ul class="list-bullet">
-          <li>
-            <p>head office <a href="http://www.dnb.co.uk/dandb-duns-number" rel="external">DUNS number</a></p>
-          </li>
-          <li>
-            <p><a href="https://beta.companieshouse.gov.uk/">Companies House number</a> (if you have one)</p>
-          </li>
-          <li>
-            contact details
-          </li>
-        </ul>
+      <div class="dmspeak">
+        <p>If your company is not <a href="https://beta.companieshouse.gov.uk/" rel="external">registered with Companies House</a>,
+           you’ll need to to <a href="https://www.dnb.co.uk/duns-number/lookup/request-a-duns-number.html" rel="external">apply for a DUNS number</a>
+           before you create a supplier account.
+        </p>
+
+        <p>
+          This will take 3 to 5 days.
+        </p>
+        <br />
+        <p>When you have set up your account we will send you a confirmation email. You must click on the link in the
+           email to activate your supplier account.</p>
       </div>
+      <br />
```

Comment:
A `<br/>` after a block element? Would be surprised if this had any effect.

Reply:
Oh it does. Always thought superfluous `<br/>`s got squashed out.

Reply:
With the `<br />`:
![screen shot 2017-09-04 at 10 50 16](https://user-images.githubusercontent.com/6525554/30021327-eaf638bc-915e-11e7-9350-5427292f959b.png)

Without the `<br />`:
![screen shot 2017-09-04 at 10 50 04](https://user-images.githubusercontent.com/6525554/30021332-ee14a380-915e-11e7-8745-d272cd10179f.png)

Reply:
Whether this is the "right" way to do this is another matter completely :)